### ISSUE TITLE
Allow zero forced convection constants 

### ISSUE THREAD

**Issue by shostikk** [firemodels/fds#6340]
Setting 
	  C_FORCED_CONSTANT=0
	  C_FORCED_RE=0
turns on the default mode of forced convection coefficient calculation. 

Simple solution is to document this as a feature in User's guide, but IMO it would better to allow the user to turn off forced convection this way. This requires initializing the parameters to a negative value, and changing the logic in read.f90.

Version          : FDS 6.6.0
Revision         : FDS6.6.0-131-g88ae75a-HEAD

**Comment by mcgratta** [comment]
If this does not change functionality, please do this now because we are preparing for a minor release. Thanks.

**Comment by shostikk** [comment]
What is the recommended way of figuring out if user has specified a real number (EB) that can have both negative and positive values?

IF (C_FORCED_CONSTANT > -HUGE(EB))  ???

**Comment by mcgratta** [comment]
That's fine, but I usually use something like 1.E10 as a big number.

**Comment by rmcdermo** [comment]
Sorry if this is a dumb question, but why would the coefficient ever be negative?

**Comment by shostikk** [comment]
Usually not. But isnt that basically just a two-term polynomial for Nu. User could have a dependency btw Nu, Re and Pr where the constant is actually negative?

**Comment by rmcdermo** [comment]
I can't think of such a case.  If you look at any Nu vs. Ra curve that I can find, it looks like it is monotonically increasing.  I think that unless we can show there is an empirical correlation that uses a negative coefficient, we should constraint our constants to be >= 0.

**Comment by shostikk** [comment]
Fine with that. Will do

**Comment by shostikk** [comment]
Two more problems:

1) We set NUSSELT = MAX(1._EB, C_FORCED_CONSTANT + ...)
Thus, we enforce Nusselt number greater than one. This is probably true for forced convection, but prevents user from having full control on the correlation. Is this limiting important for us? 

2)   We also set H_NATURAL = MAX(H_NATURAL,2._EB*K_G/DN)
This is an undocumented feature, which basically prevents user from fully controlling natural convection through C_HORIZONTAL and/or C_VERTICAL. I can understand this may be needed for converging LES when dx gets small.
